src/main/blackbox/blackbox.c

```diff
@@ -119,7 +119,7 @@ static const char blackboxHeader[] =
     "H Product:Blackbox flight data recorder by Nicholas Sherlock\n"
     "H Data version:2\n";
 
-static const char* const blackboxFieldHeaderNames[] = {
+static const char *const blackboxFieldHeaderNames[] = {
```

Comment:
This is strange. `char *p` if ok, but `const` is type qualifier, I'd expect space before it